hack/Makefile

```diff
@@ -14,7 +14,7 @@ generate-python-client:
 	./hack/generate.sh generate_python_client
 
 generate-mocks:
-	find . -name 'mock_*.go' -type f -delete
+	find . -name 'mock_*.go' -type f -not -path './vendor/*' -delete
```

Comment:
with ``api/vendor`` and ``models/vendor`` shouldn't it be updated now?

Reply:
The mock files were not deleted from `api/vendor` and `models/vendor` hence no need to exclude here